pipeline/pipeline_tasks/parse/00_04_01_usps_zips.sql

```diff
@@ -0,0 +1,583 @@
+drop table if exists stg_analytics.sba_sfdo_zips;
+
+create table stg_analytics.sba_sfdo_zips
+(
+zip_code text,
+zip_city text,
+zip_state text,
+zip_county text
+);
+
+insert into stg_analytics.sba_sfdo_zips
+values (
```

Comment:
@makfan64 Be careful here, this doesn't return what you think this returns. When using the enclosing parentheses in values you're denoting different rows. Inside those parentheses you're denoting different columns. For example,

```
values (
('a', 'b'),
('c', 'd')
)
```
will actually return one row with two columns:
```
column1 column2
('a', 'b')   ('c', 'd')
```
However, 

```
values
('a', 'b'),
('c', 'd')
```
Returns two columns with two rows
```
column1 column2
'a'           'b'
'c'           'd'
```
which is what you actually intend here.

Long story short, I would remove the initial enclosing parentheses here.